### ISSUE TITLE
Blurry

### ISSUE THREAD

**Issue by Noggog** [paulo-roger/flying-tokens#4]
Might just be the TokenFX and how they work, but I see a noticeable blur of the token's image when I have flying mode on.

![firefox_JzasYYB4yF](https://user-images.githubusercontent.com/24981326/192048684-1ce61cf6-e760-4d47-95cc-3a88f94ce729.gif)

Might be a little harder to see it with the GIF artifacts, but all the feather edges get blurred.  Not sure if that's something in your control, but figured I'd check with you first

**Comment by paulo-roger** [comment]
This blur is a feature lol. I will add a setting to opt it off in the next release later today at best or tomorrow morning at worst.

edit: typo.

**Comment by Noggog** [comment]
Haha, awesome and thanks!   Yeah, I think if it was a more pronounced blur it would be seen as intentional.  Might just be me, but when it's a very small blur difference like that my brain interprets it as computer fuzz rather than immersion depth of field or something.   Thanks for the setting!

**Comment by paulo-roger** [comment]
You are right, I thought you were referring to the twist filter effect that sometimes seems like a little blurry. But it was part of the movement effect, if you disable it the blur disappears.

Edit: sorry for the delay, had some stuff during the weekend :S

**Comment by paulo-roger** [comment]
For while you can remove the movement in the module settings, for the next version I will add a more granular settings for each effect so the user can play with it and choose what is better.
